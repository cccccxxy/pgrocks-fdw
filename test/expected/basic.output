--
-- Test basic select, insert, delete, update operations
--
\c kvtest
You are now connected to database "kvtest" as user "postgres".
CREATE FOREIGN TABLE student(id INTEGER, name TEXT) SERVER kv_server;
CREATE FOREIGN TABLE
INSERT INTO student VALUES(20757123, 'Rafferty');
INSERT 0 1
SELECT * FROM student;  
 id        |  name  
-----------+---------
 20757123  | Rafferty
(1 row)

INSERT INTO student VALUES(20777345, 'Heisenberg');
INSERT 0 1
SELECT * FROM student;  
    id      |  name   
------------+----------
 20757123   | Rafferty
 20777345   | Heisenberg
(2 rows)

DELETE FROM student WHERE id=20777345;  
DELETE 1
SELECT * FROM student;  
    id      |  name   
------------+----------
 20757123   | Rafferty
(1 row)

UPDATE student SET name='Jones' WHERE id=20757123; 
UPDATE 1
SELECT * FROM student;  
    id      |  name   
------------+----------
 20757123   | Jones
(1 row)

DROP FOREIGN TABLE student;  
DROP FOREIGN TABLE
