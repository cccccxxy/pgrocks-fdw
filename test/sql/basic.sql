--
-- Test basic select, insert, delete, update operations
--

\c kvtest

CREATE FOREIGN TABLE test(key TEXT, value TEXT) SERVER kv_server;  

INSERT INTO test VALUES('YC', 'VidarDB');  
SELECT * FROM test;  

INSERT INTO test VALUES('California', 'Waterloo');  
SELECT * FROM test;  

DELETE FROM test WHERE key='California';  
SELECT * FROM test;  

UPDATE test SET value='VidarSQL';  
SELECT * FROM test;  

DROP FOREIGN TABLE test;  
