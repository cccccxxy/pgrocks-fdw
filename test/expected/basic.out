--
-- Test basic select, insert, delete, update operations
--
CREATE FOREIGN TABLE test(key TEXT, value TEXT) SERVER kv_server;
CREATE FOREIGN TABLE
INSERT INTO test VALUES('YC', 'VidarDB');
INSERT 0 1
SELECT * FROM test;
 key |  value
-----+---------
 YC  | VidarDB
(1 row)

INSERT INTO test VALUES('California', 'Waterloo');
INSERT 0 1
SELECT * FROM test;
    key     |  value
------------+----------
 YC         | VidarDB
 California | Waterloo
(2 rows)

DELETE FROM test WHERE key='California';
DELETE 1
SELECT * FROM test;
 key |  value
-----+---------
 YC  | VidarDB
(1 row)

UPDATE test SET value='VidarSQL';
UPDATE 1
SELECT * FROM test;
 key |  value
-----+----------
 YC  | VidarSQL
(1 row)

DROP FOREIGN TABLE test;
DROP FOREIGN TABLE
